src/main/java/seedu/taassist/logic/commands/result/CommandResult.java

```diff
@@ -0,0 +1,126 @@
+package seedu.taassist.logic.commands.result;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Objects;
+
+import seedu.taassist.logic.commands.actions.StorageAction;
+import seedu.taassist.logic.commands.actions.UiAction;
+import seedu.taassist.logic.commands.exceptions.CommandException;
+import seedu.taassist.logic.commands.exceptions.StorageActionException;
+import seedu.taassist.storage.Storage;
+
+/**
+ * Represents the result of a command execution.
+ */
+public class CommandResult {
+
+    public static final UiAction NO_UI_ACTION = UiAction.UI_NO_ACTION;
```

Comment:
`UiAction.UI_NO_ACTION` is already a constant. Would it be better to use that directly?